engine/access/rest/websockets/controller.go

```diff
@@ -108,6 +125,11 @@ func (c *Controller) readMessage() (json.RawMessage, error) {
 	if err := c.conn.ReadJSON(&message); err != nil {
 		return nil, fmt.Errorf("error reading JSON from client: %w", err)
 	}
+
+	if message == nil {
+		return nil, ErrEmptyMessage
```

Comment:
Please expand godoc here with `Expected error returns during normal operations`

Reply:
I'm pretty sure `message` can't be nil in this case. It was a workaround for tests. Looks redundant now, so I'm removing it